sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj

```
@@ -7,10 +7,12 @@
 		<PackageId>Microsoft.Azure.Management.Peering</PackageId>
 		<Description>Microsoft Azure Management Peering library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Peering</AssemblyName>
-		<Version>2.0.1-preview</Version>
+		<Version>1.0.0</Version>
```

```suggestion
		<Version>2.1.0</Version>
```
We do not decrease versions according to our versioning guide: https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md
Also this is feature addition to 2.0.1-preview.

---

Please update the versions in `AssemblyInfo.cs` as well

---

Updated.